cypress/e2e/inventory/search/browse-call-number-with-space.cy.js

```diff
@@ -34,21 +39,29 @@ describe('ui-inventory: search', () => {
   beforeEach('Creating user and instance with item with call number', () => {
     cy.getAdminToken().then(() => {
       InventoryInstances.createInstanceViaApi(item.instanceName, item.itemBarcode, item.publisher, item.holdingCallNumber, item.itemCallNumber);
+      cy.createTempUser([
+        permissions.uiCallNumberBrowse.gui,
+        permissions.inventoryAll.gui
+      ]).then(userProperties => {
+        testData.user = userProperties;
+        cy.login(userProperties.username, userProperties.password, { path: TopMenu.inventoryPath, waiter: InventorySearchAndFilter.waitLoading });
+      });
     });
-
-    cy.loginAsAdmin({ path: TopMenu.inventoryPath, waiter: InventorySearchAndFilter.waitLoading });
   });
 
   afterEach('Deleting user and instance', () => {
     InventoryInstances.deleteInstanceAndHoldingRecordAndAllItemsViaApi(item.itemBarcode);
+    Users.deleteViaApi(testData.user.userId);
   });
 
   it('C358140 Verify that browsing for "call number" with "space" value will get the correct result (spitfire)', { tags: [DevTeams.spitfire, TestTypes.smoke] }, () => {
```

Comment:
test is failed, please check it

Reply:
The test is passed https://jenkins-aws.indexdata.com/job/Testing/job/Cypress%20tests/526/allure/